src/components/ItemList/data.tsx

```diff
@@ -0,0 +1,63 @@
+const BEERS_PER_PAGE: number = 10;
+const DEFAULT_PAGE: number = 1;
+const ROOT_ENDPOINT: string = 'https://api.punkapi.com/v2/';
+const INITIAL_ENDPOINT: string = `beers?page=${DEFAULT_PAGE}&per_page=${BEERS_PER_PAGE}`;
+const BEER_NAME: string = '&beer_name=';
+const EMPTY_ITEMS_ARRAY: IBeerDetails[] = [
+  {
+    id: 0,
+    name: '',
+    tagline: '',
+    first_brewed: '',
+    description: '',
+    image_url: '',
+    abv: 0,
+    ibu: 0,
+    target_fg: 0,
+    target_og: 0,
+    ebc: 0,
+    srm: 0,
+    ph: 0,
+    attenuation_level: 0,
+    volume: {},
+    boil_volume: {},
+    method: {},
+    ingredients: {},
+    food_pairing: [''],
+    brewers_tips: '',
+    contributed_by: '',
+  },
+];
+
+export interface IBeerDetails {

```

Comment:
Keep types/interfaces in one folder -> files

Reply:
Fixed in https://github.com/ylakhtin/rs-react-components/commit/5f4a7d6fe4fea2271c6d9148c8e939e4cd5c4946